src/js/apps/patients/worklist/filters_app.js

```diff
@@ -0,0 +1,74 @@
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { FiltersView, GroupsDropList, ClinicianDropList, ClinicianClearButton } from 'js/views/patients/worklist/worklist_views';
+
+export default App.extend({
+  onBeforeStart(options) {
```

Comment:
probably a decent case for `this.mergeOptions(options, ['currentClinician', 'groupId', 'clinicianId']);`